style.css

```diff
@@ -0,0 +1,114 @@
+#bgimage {
+  position: absolute;
+  background-image: url("images/Illustration.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  padding-left: 165%;
+  padding-top: 900px;
+  z-index: 1;
+}
+
+.navigation {
+  position: absolute;
+  display: flex;
+  background: #3c3a39;
+  align-items: center;
+  text-align: center;
+  width: 100%;
+  height: 92px;
+  left: 0;
+  top: 0;
+}
+
+#name {
+  padding-left: 24px;
+  top: 10px;
+  width: 77px;
+  height: 24px;
+  font-family: 'Roboto', Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.body {
+  position: absolute;
+  display: flex;
+  background: #1c1a19;
+  flex-direction: column;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+  left: 0;
+  top: 92px;
+  padding: 120px, 24px, 120px, 24px;
+}
+
+#one {
+  margin-bottom: -6px;
+  margin-top: -6px;
+  height: 124px;
+  width: 327px;
+  padding-left: 24px;
+  padding-top: 120px;
+  font-family: 'Crete Round', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-size: 56px;
+  font-weight: 400;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  text-align: left;
+  color: #ff6b00;
+}
+
+#two {
+  margin-bottom: -6px;
+  margin-top: -6px;
+  height: 27px;
```

Comment:
Great work @StarMindz 🤝

- [ ] [Optional] It is advisable to always use class selectors instead of IDs to select multiple properties.